README.md

```diff
@@ -189,7 +189,7 @@ $ colcon release-in-container --help
 
 usage: colcon release-in-container [-h] [--ros-distro ROS_DISTRO] [--bloom-generator {debian,rosdebian}] [--debug] [--shell-after] [--paths [PATH [PATH ...]]]
 
-Call a colcon build command inside a fresh container.
+Call a bloom generate command inside a fresh container.
```

Comment:
It's not only 'bloom-generate' is it?

Reply:
No, it also launched the `fakeroot debian/rules binary` which is actually building the deb.
I wanted to have a sentence that look like a colcon command.
What if instead I put:
```suggestion
Generate Debian package inside a fresh container.
```
?

Reply:
Yep that's better :+1: 
You can even mention the tools e.g.: `Generate Debian package inside a fresh container using bloom and fakeroot`